content/libraries/php.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: PHP
+description: Official Top.gg PHP library
+---
+
+# PHP Library
+
+This is our official PHP Library for top.gg, if you have any issues/bugs please submit an issue on our github.
+
+- [GitHub Link](https://github.com/vapourlabs/TopPHP)
+
+(I cannot make a repo for the actual /top-gg/php-sdk version, so a clone will be req.)
```

Comment:
This doesn't need to be in the docs

Reply:
I can make a packagist upload. Will update link ASAP to reflect.